bin/maintengine.py

```diff
@@ -24,17 +24,29 @@
 
 import os.path
 import time
+import logging
+
 import nav.path
-from nav.maintengine import init_logging
+from nav.logs import init_generic_logging
 from nav.maintengine import check_devices_on_maintenance
 
 
+LOG_FORMAT = "[%(asctime)s] [%(levelname)s] [pid=%(process)d %(name)s] %(message)s"
```

Comment:
Is this vastly different from the "standard" log format, or is it just the pid number? Not sure why we would need  a pid number here, as maintengine is usually a single process. It could be that it was added to debug issues where the maintengine script spent more than 5 minutes on its working, causing the next cronjob to start up before the first one was finished...